build-tests/x86/test-image-iso/appliance.kiwi

```diff
@@ -31,7 +31,7 @@
         <package name="bind-utils"/>
         <package name="patterns-openSUSE-base"/>
         <package name="systemd"/>
-        <package name="plymouth-branding-openSUSE"/>
+        <package name="plymouth-branding-bgrt"/>
```

Comment:
Do we have a `plymouth-branding-bgrt` package? I thought we still have `plymouth-branding-openSUSE`? We also need `plymouth-theme-bgrt` in the package set.

Reply:
No we don't have one, I'm just stupid :) Updated the PR

I changed the iso build test to use this branch for building and it succeeded:

* https://build.opensuse.org/package/show/Virtualization:Appliances:Images:Testing_x86/test-image-iso